pkg/cmd/cli/cmd/newapp.go

```diff
@@ -166,6 +166,8 @@ func RunNewApplication(fullName string, f *clientcmd.Factory, out io.Writer, c *
 			}
 		}
 	}
+	fmt.Fprintln(c.Out(), "Run 'oc status' to view your app.")
```

Comment:
Well, I am not hot about it either but for consistency we should return suggestions based on what a user run. So if someone hits `openshift cli new-app https://theirrepo/theirsource` give them `openshift cli` suggestions. We already do that in our examples.